packages/IMAPClient-Core.package/ICCommand.class/class/resetFlag.for..st

```diff
@@ -0,0 +1,4 @@
+commands
+resetFlag: aFlagString for: aMessageId
+
+	^ self join: {'STORE' . aMessageId . '-FLAGS \', aFlagString withFirstCharacterUppercase}
```

Comment:
Before the refactor this parts predecessor contained a check whether `aFlagString` was a valid one:
```
(aFlag == #flagged) ifTrue: [self sendCommand: (self class resetFlagFlagged: anEmail messageID)].
	(aFlag == #seen) ifTrue: [self sendCommand: (self class resetFlagSeen: anEmail messageID)].		
```

That is not the smartest thing to do obviously but how do we check whether the string correctly describes a existing flag?
Or do we allow bad strings and the endpoint is handling the servers's error response?

Reply:
This is a good point. I thought that there should never be invalid flags, because they are constants in the IMAP standard and therefor we don't need a validation. But a check would help other developers finding errors faster. I will add a validation check in `ICCommand`